app/src/main/java/io/gnosis/safe/ui/base/Adapter.kt

```diff
@@ -59,4 +59,3 @@ abstract class Adapter<T, VH : Adapter.ViewHolder<T>> : RecyclerView.Adapter<VH>
 }
 
 class UnsupportedViewType(message: String? = null) : Throwable(message)
-
```

Comment:
Don't worry, there is still a new line at the end of the file. This removes an additional newline (there were two).